### ISSUE TITLE
when release the yolo26?

### ISSUE THREAD

**Issue by xinsuinizhuan** [ultralytics/ultralytics#22334]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

when release the yolo26?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @xinsuinizhuan, thank you for your interest in Ultralytics 🚀! This is an automated response — an Ultralytics engineer will review and assist you here soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
Thanks for the question—YOLO26 isn’t released yet; it’s currently in preview. ([docs.ultralytics.com](https://docs.ultralytics.com/models/yolo26/)) Please watch the [Ultralytics Releases](https://github.com/ultralytics/ultralytics/releases) and follow the [YOLO26 preview page](https://docs.ultralytics.com/models/yolo26/) for updates; for production today, use [YOLO11](https://docs.ultralytics.com/models/yolo11/). ([github.com](https://github.com/ultralytics/ultralytics/releases))

**Comment by ElliottDyson** [comment]
> Thanks for the question—YOLO26 isn’t released yet; it’s currently in preview. ([docs.ultralytics.com](https://docs.ultralytics.com/models/yolo26/)) Please watch the [Ultralytics Releases](https://github.com/ultralytics/ultralytics/releases) and follow the [YOLO26 preview page](https://docs.ultralytics.com/models/yolo26/) for updates; for production today, use [YOLO11](https://docs.ultralytics.com/models/yolo11/). ([github.com](https://github.com/ultralytics/ultralytics/releases))

I am also very much interested in YOLOv26, is there a way to subscribe to the updates, seeing as we have no timeframe for updates to the preview page? I'm particularly looking forward to the new MuSGD optimiser, as well as ProgLoss and STAL.

Thank you.

**Comment by glenn-jocher** [comment]
Thanks—please Watch this repo (Custom > Releases) for notifications and follow our public channels for YOLO26 news; updates are posted on the YOLO Vision 2025 page and the Meet Ultralytics YOLO26 blog post, where we’ll share details on MuSGD, ProgLoss, and STAL when ready: [YOLO Vision 2025](https://www.ultralytics.com/events/yolovision), [Meet Ultralytics YOLO26](https://www.ultralytics.com/blog/meet-ultralytics-yolo26-a-better-faster-smaller-yolo-model).

**Comment by xinsuinizhuan** [comment]
when release？

**Comment by glenn-jocher** [comment]
YOLO26 is still in development with no date to share; we’ll update this issue when downloads go live—meantime, the latest public status recap is in our [YOLO Vision 2025 Shenzhen highlights](https://www.ultralytics.com/blog/ultralytics-key-highlights-from-yolo-vision-2025-shenzhen).

**Comment by IncubatorShokuhou** [comment]
When will MuSGD be available?

**Comment by glenn-jocher** [comment]
MuSGD will ship together with YOLO26 training support in a future `ultralytics` release; there’s no public timeline to share yet, but when it’s available it will be documented under the optimizer options in the [Ultralytics train mode docs](https://docs.ultralytics.com/modes/train/).

**Comment by xinsuinizhuan** [comment]
when to release yolov26? 2026 soon coming!

**Comment by glenn-jocher** [comment]
We haven’t announced a YOLO26 release date yet; for the fastest notification when it’s actually shipped (including MuSGD), please enable GitHub “Watch → Releases” for this repo as described in [Watching repositories for releases](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications#about-watching-and-unwatching-repositories), and if you want ongoing status Q&A please post in [Ultralytics Discussions](https://github.com/orgs/ultralytics/discussions) for visibility.
